### DISCUSSION TITLE
indication of release dates (e.g. RC2 or otherwise)

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#1098] [General]
i know, i know, it's ready when it's ready. i respect that and wouldn't have it any other way. but, it would be really helpful to know when a pending release is scheduled to be issued, such as RC2, which seems imminent but so did RC1 for a week or two. It wouldn't surprise me if people are holding back on bug reports (i know i am) so that we can get a release in our hands so that we're not just battling ancillary issues that have already been resolved.

as an aside, i would welcome this as a feature offered within some tier of the licensing model - the right to know about road maps, release dates, features in the pipeline, things like that.

**Comment by meetmandeep** [comment]
@eswanzey I hear you loud and clear. It's been a sprint for us from past 2 years to get Vanjaro to official release and we're almost there. RC 2 is ready but pending internal QA approval. I'm hopeful it will be cleared for release by end of this week. 

We'll soon open up invites for a future Partners Program with a dedicated channel for bi-directional feedback on roadmaps and release dates. 

We're also considering adding a monthly webinar for all stakeholders for one-on-one questions; let me know if that would help as well.

**Reply by eswanzey** [reply]
Thanks @meetmandeep for your reply.

A monthly webinar would be fantastic. Because it would be for all stakeholders, it would be an excellent way to check the overall pulse of the Vanjaro user base - useful for you, useful for Partners, useful for end users - I look forward to attending those.

I'm counting on a Partners Program, so good news there as well.

For a number of reasons, I know that I will participate in all of the above, sight unseen, as far as costs are concerned. If you were to have details on any of these things that you could send me, please do so.

**Reply by meetmandeep** [reply]
Great. We'll have our first webinar next month and details will follow shortly.

**Reply by turbz214** [reply]
Any update on timing? We have a couple clients that want to go live ASAP, so we would like to give them a realistic date.

**Reply by meetmandeep** [reply]
RC 2 is now available. Please join us for our first webinar on May 19th where we'll discuss the roadmap and official release date: https://www.vanjaro.com/resources/webinars

**Reply by joshgk00** [reply]
Are there any updates for the official 1.0 release?

**Reply by scottw12** [reply]
Or even a RC3. Seems to be very quiet both here and in issues.  I sure hope all is well with the Vanjaro Team.

**Reply by eswanzey** [reply]
I'm pretty sure that when it does release, along with the docs and services, some among us will cry tears of joy!

**Reply by eswanzey** [reply]
but yeah..."are we there yet?" I hope that phrase translates appropriately across cultures.

**Reply by meetmandeep** [reply]
Almost there! We're just waiting for the go ahead from marketing team. I don't want to commit to a date and not able to meet it; but anytime now.
